plugins/modules/packaging/language/pear.py

```diff
@@ -51,6 +57,30 @@
     name: pecl/json_post
     state: present
 
+# Install pecl package with expected prompt
```

Comment:
It's better to use task names than comments:
```suggestion
- name: Install pecl package with expected prompt
```